Userland/Libraries/LibWeb/HTML/HTMLTableElement.cpp

```diff
@@ -221,11 +228,17 @@ void HTMLTableElement::delete_t_foot()
     }
 }
 
+// https://html.spec.whatwg.org/multipage/tables.html#dom-table-tbodies
```

Comment:
These properties are also missing [SameObject] in the IDL. https://github.com/SerenityOS/serenity/blob/c922d21ecb64bf487b3331734e4e679bfe3fcd0b/Userland/Libraries/LibWeb/HTML/HTMLTableElement.idl#L23